### ISSUE TITLE
iperf3 test reports error on last interval when running UDP test on high latency network

### ISSUE THREAD

**Issue by baltshul** [esnet/iperf#718]
_NOTE: The iperf3 issue tracker is for registering bugs, enhancement
requests, or submissions of code.  It is not a means for asking
questions about building or using iperf3.  Those are best directed
towards the iperf3 mailing list at iperf-dev@google-groups.com or
question sites such as Stack Overflow
(http://www.stackoverflow.com/).  A list of frequently-asked questions
regarding iperf3 can be found at http://software.es.net/iperf/faq.html._

# Context

* Version of iperf3: 3.2 and above

* Hardware: any

* Operating system (and distribution, if any): Linux

* Other relevant information (for example, non-default compilers,
  libraries, cross-compiling, etc.):

_Please fill out one of the "Bug Report" or "Enhancement Request"
sections, as appropriate._

# Bug Report
When running iperf3 UDP test on high latency network, the last reporting interval doesn't complete and prints
iperf3: error - control socket has closed unexpectedly

* Expected Behavior
should complete test on high latency network even with > 5s latency

**Comment by baltshul** [comment]
Problem seen with TCP traffic test on high latency network as well.

**Comment by bmah888** [comment]
I'm wondering if this problem could be another aspect of this bug, fixed in iperf 3.4:
```
  * The client no longer closes its control connection before sending
    end-of-test statistics to the server (#677). This fixes a
    regression introduced in iperf-3.2.
```
Have you tried with iperf 3.4 or later?  (3.5 is current as of this writing.)

**Comment by bmah888** [comment]
No feedback after five months, closing.
